src/cls/rgw/cls_rgw_types.h

```diff
@@ -1045,7 +1045,7 @@ WRITE_CLASS_ENCODER(cls_rgw_gc_obj_info)
 
 struct cls_rgw_lc_obj_head
 {
-  time_t start_date;
+  time_t start_date = 0;
```

Comment:
@joscollin Yes It looks..  It would be system's current time.
Why?
Since `void encode(bufferlist& bl)` calls `::encode(t, bl);` with start_time, I believe encode would not start on 0, It would start on Current time.

But It would be good to take 2nd opinion from any other reviewer as well...